packages/block-library/src/image/transforms.js

```diff
@@ -59,6 +59,7 @@ const schema = ( { phrasingContentSchema } ) => ( {
 			...imageSchema,
 			a: {
 				attributes: [ 'href', 'rel', 'target' ],
+				classes: [ /[\w-]*/ ],
```

Comment:
Maybe we should support a wildcard matcher in `cleanNodeList` for `class`. There's already support for it for `children` property.

https://github.com/WordPress/gutenberg/blob/467e4a22b6875aef32c6baf09ef38f62eeba2edd/packages/dom/src/dom/clean-node-list.js#L109-L112

Reply:
Here's the follow-up #67830.